src/quacc/recipes/common/phonons.py

```diff
@@ -31,6 +32,7 @@ def phonon_subflow(
     atoms: Atoms,
     force_job: Job,
     relax_job: Job | None = None,
+    fixed_atoms: list[int] | None = None,
```

Comment:
Do we want to come back to this approach?

Reply:
@tomdemeyere: Ah, we did that before. Sorry for the memory issues..........

Let me read up on what happened before.